scripts/backup/pg_restore.sh

```diff
@@ -49,8 +47,12 @@ if [ "$BACKUP_USER" != "" -a "$(id -un)" != "$BACKUP_USER" ]; then
 fi;
 
 if [ -z $RESTORE_DIR ]; then
-	echo "[ERROR]["$(date +\%Y-\%m-\%d\ %H:%M:%S:%3N)"] Backup not provided, please use -d option." 1>&2
-	exit 1;
+    echo -e "\nRestore directory is empty, retrieving from Google Drive backup"
+    if ! gdrive download "$DATABASE".sql.gz --timeout "$TIMEOUT"; then
```

Comment:
@carlosmiranda I think will be greate if we set a variable for path where download to, WDYT?

Reply:
@vladarefiev done